glean-core/uniffi/src/glean.udl

```diff
@@ -10,6 +10,10 @@ namespace glean {
     void glean_set_experiment_active(string experiment_id, string branch, record<DOMString, string> extra);
     void glean_set_experiment_inactive(string experiment_id);
     RecordedExperiment? glean_test_get_experiment_data(string experiment_id);
+
+    boolean glean_set_debug_view_tag(string tag);
+    boolean glean_set_source_tags(sequence<string> tags);
+    void glean_set_log_pings(boolean value);
```

Comment:
I guess it's been this way for a while, but it seems weird that both of the other `set` functions return the boolean and the `log_pings` doesn't.

Reply:
both the others validate the tags, at which point we bail out further processing if they are wrong.
`log_pings` has nothing to validate.